GameServer/Extensions.cs

```diff
@@ -42,6 +43,52 @@ public static void fill(this BinaryWriter list, byte data, int length)
             }
         }
     }
+
+    static class EnumParser
+    {
+        public static SummonerSpellIds ParseSummonerSpell(string spellName)
+        {
+            if (spellName == "FLASH")
```

Comment:
The implementation was just moved to it's own place (before it was on Game.cs). I definitely get the idea though, and re-implemented it with a Dictionary in my next commit.